src/components/WalletList/WalletList.tsx

```diff
@@ -144,7 +159,35 @@ export const WalletList = ({ wallets, onSelect, showHeader, calculateMargin }: P
             {translateRaw('ADD_ACCOUNT_IMPORT_SETTINGS_LINK')}
           </Link>
         </Info>
+        <SDemoButtonContainer>
+          <Link to={ROUTE_PATHS.DASHBOARD.path}>
+            <Button
+              colorScheme={TButtonColorScheme.warning}
+              disabled={isDemoMode}
+              onClick={() => importState(JSON.stringify(DEMO_SETTINGS))}
```

Comment:
Should this be available if you already have added accounts? I doubt many would want to use it again.

Reply:
ah. nice catch. Ill probably just hide the button if added accounts > 0.